spinalcordtoolbox/types.py

```diff
@@ -771,7 +771,7 @@ def display(self, mode='absolute'):
         # ax.set_aspect('equal')
         plt.xlabel('z')
         plt.ylabel('x')
-        ax = plt.subplot(212)
+        ax = plt.subplot(212)  # noqa: F841
```

Comment:
Is the reason these got `noqa` comments because of the commented out `# ax.set_aspect('equal')`? (i.e. wanting to allow that line to be uncommented without throwing an error?)